cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,159 @@
 describe('Web Tables page', () => {
-  it('', () => {
 
+  const newWorker = {
+    firstName: 'Lilith',
+    lastName: 'Mother',
+    email: 'liliththemother@mail.com',
+    age: 18,
+    salary: 10000,
+    department: 'Testing'
+  }
+
+  const worker = {
+    firstName: 'Kierra',
+    lastName: 'Gentry',
+    email: 'kierra@example.com',
+    age: 29,
+    salary: 2000,
+    department: 'Legal'
+  }
+
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  it('should check pagination', () => {
+    cy.get('.-pagination')
+      .should('contain', 'Previous')
+      .and('contain', 'Next');
+    cy.get('.-pageInfo')
+      .should('contain', 'Page');
+  });
+
+  it('should cleck rows count selection ', () => {
+    cy.get('select')
+      .select('5 rows')
+    cy.get('select')
+      .should('contain', '5 rows')
+  });
+
+  it('should provide an ability to add a new worker', () => {
+    cy.get('#addNewRecordButton')
+      .click();
+    cy.get('#firstName')
+      .type(newWorker.firstName);
+
+    cy.get('#lastName')
+      .type(newWorker.lastName);
+
+    cy.get('#userEmail')
+      .type(newWorker.email);
+
+    cy.get('#age')
+      .type(newWorker.age);
+
+    cy.get('#salary')
+      .type(newWorker.salary);
+
+    cy.get('#department')
+      .type(newWorker.department);
+
+    cy.get('#submit')
+      .click();
+
+    cy.get('.rt-tr-group')
+      .should('contain', newWorker.firstName)
+      .and('contain', newWorker.lastName)
+      .and('contain', newWorker.email)
+      .and('contain', newWorker.salary)
+      .and('contain', newWorker.age)
+      .and('contain', newWorker.department)
+  });
+
+  it('should provide an ability to delete worker', () => {
+    cy.get('#delete-record-1')
+      .should('exist');
+
+    cy.get('#delete-record-1')
+      .click();
+
+    cy.get('.rt-tbody')
+      .should('not.contain.html', '#delete-record-1');
+  });
+
+  it('should provide an ability to delete all workers', () => {
```

Comment:
you have to be able to delete all workers no matter how many of them are in the table. try to implement a loop here